bpython/curtsies.py

```diff
@@ -196,16 +245,20 @@ def curtsies_arguments(parser):
             with repl.window as win:
                 with repl:
                     repl.height, repl.width = win.t.height, win.t.width
-                    exit_value = repl.mainloop(True, paste)
+                    repl.mainloop(True, paste)
```

Comment:
Is this a change in behavior?

Reply:
`FullCurtsiesRepl.mainloop()` never returns a non-None value, so this is not a change.